class.two-factor-core.php

```diff
@@ -217,7 +217,7 @@ function login_form_twostep() {
 				return;
 			}
 
-			$this->login_html( $user, $login_nonce, $_REQUEST['redirect_to'], __( 'ERROR: Invalid verification code.', 'two-factor' ) );
+			$this->login_html( $user, $login_nonce, $_REQUEST['redirect_to'], esc_html__( 'ERROR: Invalid verification code.', 'two-factor' ) );
```

Comment:
This is escaped inside `login_html()` here: https://github.com/georgestephanis/two-factor/blob/41bed06cc04ec0683ce34ac826cc8a29dd2e5790/class.two-factor-core.php#L144 -- doesn't need double escaping.